ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -304,7 +327,7 @@ public enum EnabledOverride {
         nameMap.put(
             vectorizationEnabledOverride.name().toLowerCase(), vectorizationEnabledOverride);
```

Comment:
could you please create toString() method that returns name().toLowerCase() and use that

Reply:
I haven't introduced that enum but since we are improving slightly over the existing I have done that.